app/views/member_mailer/welcome_coach.html.haml

```diff
@@ -6,7 +6,11 @@
 
 %p
   - if @member.subscriptions.any?
-    You'll automatically receive invitations to our workshops via email.
+    For our London chapter, you will be able to RSVP to workshops via our website. We open RSVPs at 1PM on the Friday before the workshop. You will be able to register through the workshop page. Join the #london-chapters channel
```

Comment:
I think this is good, but here's how I'd word it (if it were up to me). Up to you which you think is better:

> If you're subscribed to our London chapter, you won't receive workshop invitations via email. We open RSVPs at around 1pm on the Friday before the workshop, and you will be able to register through the workshop page on our website. To be notified when we open RSVPs, join the #london-chapters channel [on our Slack](http://codebar-slack.herokuapp.com/), or [follow us on Twitter](https://twitter.com/codebar).
If you are subscribed to any of our other chapters, you'll automatically receive email invitations to our workshops as soon as a workshop is open for RSVP.